alembic/versions/46e06e80c43a_add_shops_table.py

```diff
@@ -0,0 +1,51 @@
+"""add shops table
+
+Revision ID: 46e06e80c43a
+Revises: 0f7e921cbc13
+Create Date: 2020-05-15 13:15:59.526824
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+import geoalchemy2
+
+
+# revision identifiers, used by Alembic.
+revision = '46e06e80c43a'
+down_revision = '0f7e921cbc13'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('shops',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=50), nullable=False),
+    sa.Column('phone_number', sa.String(length=15), nullable=True),
+    sa.Column('address', sa.String(length=50), nullable=True),
+    sa.Column('owner_id', sa.Integer(), nullable=False),
+    sa.Column('location', geoalchemy2.types.Geography(geometry_type='POINT', srid=4326), nullable=True),
+    sa.ForeignKeyConstraint(['owner_id'], ['owner.id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_index(op.f('ix_shops_id'), 'shops', ['id'], unique=False)
+    # op.drop_table('spatial_ref_sys')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('spatial_ref_sys',
+    sa.Column('srid', sa.INTEGER(), autoincrement=False, nullable=False),
+    sa.Column('auth_name', sa.VARCHAR(length=256), autoincrement=False, nullable=True),
+    sa.Column('auth_srid', sa.INTEGER(), autoincrement=False, nullable=True),
+    sa.Column('srtext', sa.VARCHAR(length=2048), autoincrement=False, nullable=True),
+    sa.Column('proj4text', sa.VARCHAR(length=2048), autoincrement=False, nullable=True),
+    sa.CheckConstraint('(srid > 0) AND (srid <= 998999)', name='spatial_ref_sys_srid_check'),
+    sa.PrimaryKeyConstraint('srid', name='spatial_ref_sys_pkey')
+    )
```

Comment:
@iamareebjamal Should I remove this? I have created `postgis` through SQL command.

Reply:
No